ci/prow-entrypoint.sh

```diff
@@ -123,10 +123,18 @@ kola_test_qemu() {
 
     # Skip Secure Boot tests on SCOS for now
     # See: https://github.com/openshift/os/issues/1237
-    if rpm-ostree compose tree --print-only "${manifest}" | jq -r '.packages[]' | grep -q "centos-release"; then
-        cosa kola --basic-qemu-scenarios --output-dir ${ARTIFACT_DIR:-/tmp}/kola-basic
+    # Due to the changes in https://github.com/coreos/coreos-assembler/pull/3652
+    # we need to check if the basic.nvme is available in the list of tests
+    if cosa kola list --json | jq -r '.[].Name' | grep -q "basic.nvme"; then
+        if ! rpm-ostree compose tree --print-only "${manifest}" | jq -r '.packages[]' | grep -q "centos-release"; then
+            cosa kola --denylist-test *.uefi-secure --output-dir ${ARTIFACT_DIR:-/tmp}/kola-basic
```

Comment:
We don't want to run the full suite twice. We should just set a variable here that has the `--denylist` flag which we pass to `cosa kola run` below.